x-pack/plugins/security_solution/public/timelines/components/timeline/helpers.tsx

```diff
@@ -184,7 +184,7 @@ export const focusUtilityBarAction = (containerElement: HTMLElement | null) => {
  * Resets keyboard focus on the page
  */
 export const resetKeyboardFocus = () => {
-  document.querySelector<HTMLAnchorElement>('header.headerGlobalNav a.chrHeaderLogo')?.focus();
```

Comment:
@elastic/security-threat-hunting-investigations in reviewing this, let's consider accessibility. The way this works should keep a logical and intuitive flow in keyboard navigation.

Reply:
@michaelolo24 / @kqualters-elastic Any historical context here? Was this just added to remove keyboard focus? 

As suggested by @tsullivan in the slack thread , I think I agree that we can use `document.body.focus()` as well. Do you have any opinion?

Reply:
I am not sure on the context of this to be honest, seems it's used in conjunction with https://github.com/elastic/kibana/blob/main/x-pack/plugins/timelines/common/utils/accessibility/helpers.ts#L772 . What exactly is wrong with this? "The way this works should keep a logical and intuitive flow in keyboard navigation." is this not the case now?

Reply:
> What exactly is wrong with this? "The way this works should keep a logical and intuitive flow in keyboard navigation." is this not the case now?

@kqualters-elastic this is what SharedUX team needs the @elastic/security-threat-hunting team to test and verify, because we don't know how to access all of the parts that this change could affect. Let us know if you or a delegate can confirm that, or wish to see this modified in some way. Thanks!

Reply:
Thanks I will test this tomorrow..

Reply:
Okay I did some desk testing with the new change and there are couple of issues with this change.

1. Although body is focusable, it is not tabbable which actually takes user out of `tab` order and if once the focus is `reset`on `body`, pressing `Tab` after that does nothing.
2. In previous case, the reset was focusing the logo, which was kind of `first` in the tab order and users can now tab through out the page which is not possible in the new change.
3. It looks like, this function ( old version, not the change) is broken in serverless, since logo with class `chrHeaderLogo` does not even exists in Serverless.
    - this begs the question, if we even need this functionality. I will discuss with the team today and if we do not, i will push the commit to remove it.

Reply:
Thanks @logeekal!

Reply:
@logeekal Does this PR need to be held on the discussion with your team? I hope not, and I think that if nothing else is blocking this PR, it could get approved and the team discussion could be handled as follow-up work. 

Does that sound ok?

Reply:
@tsullivan , Sorry for the delay from my side. It took time to get full information, but i can say that this piece of code is not required at our end. But I  will create a separate PR to remove it so that you are not blocked.

I will create a separate PR to add that code. Thanks for your patience.